content/articles/what-happens-if-i-stop-paying.markdown

```diff
@@ -0,0 +1,16 @@
+---
+title: What happens if I stop paying for my DNSimple subscription?
+excerpt: Description of what happens if an account is unsubscribed. 
+categories:
+ Account
+---
+
+# What happens if I stop paying for my DNSimple subscription?
+
+After a certain period of time, an account is cancelled after payment cannot be collected. What happens if such a thing were to happen depends on the current configuration of your delegation. 
+
+If you are pointing to DNSimple's nameservers and your account closes, **your domain will no longer resolve, and will become unreachable**. 
```

Comment:
I'm not sure the proper phrasing is "account closes", but I don't know the right alternative right now either. Maybe "subscription lapses"? It might also be good to mention that we'll respond with a DNS error code (not sure if it's `SERVFAIL` or `NXDOMAIN` though).